castervoice/rules/ccr/rust_rules/rust.py

```diff
@@ -21,9 +21,9 @@ class Rust(MergeRule):
             R(Text("else {}") + Key("left/5:3")),
         #
         SymbolSpecs.SWITCH:
-            R(Text("match")),
```

Comment:
The match command is always followed by a space much like static So I added it here